client/systems.go

```diff
@@ -140,6 +140,21 @@ func (client *Client) RebootToSystem(systemLabel, mode string) error {
 	return nil
 }
 
+type StorageEncryption struct {
+	// string: "yes"/"no-but-optional"/"no-and-required"
+	Available string `json:"available,omitempty"`
```

Comment:
maybe we should call this support?

```
// Support describes the level of hardware support available, it can have the values:
// disabled, present, absent, defective
```

instead of present/absent we could use available/unavailable

we would set this also to disabled as in that case we do no checks anyway

Reply:
Nice, I like this! I updated the code - one alternative way to describe "defective" might be "insufficient" or "deficient" or "inadequate" or "unqualified".

Reply:
defective still sounds the best to me, it's also the one that is mostly thing-oriented vs abstract/person

Reply:
Works for me, just wanted to throw some ideas around :) It's defective now in the code.